content/about/can-tor-help-against-viruses/contents.lr

```diff
@@ -0,0 +1,11 @@
+_model: question
+---
+title: Can Tor help you against viruses?
+---
+seo_slug: use-tor-against-viruses
+---
+description:
+Tor is not an Anti-virus/Malware Software.
+It is an anonymity network,and you can use the Tor browser to avoid getting tracked on the internet.
+If you use it to gain access to a web page full of malware,and your browser and OS are vulnerable to the malware on that page then yes you will be affected.
```

Comment:
This paragrah is confusing:

- [ ] what's the difference between a page "full of malware" and with another page with a "single malware"? From browser protection this makes no difference.
- [ ] when you write "your browser", that means people are using their regular browser or Tor Browser?